doc/admin.md

```diff
@@ -22,7 +22,6 @@
         -   [componentName](#componentname)
```

Comment:
As a general comment in this PR, being a mayor refactor of the code, I think it would be a good idea to mention in the CHANGES_NEXT_RELEASE file. For instance:

```
- Large refactor of IOTA Lib code to make it simpler
```

or something similar.

Probably this deserves a new mayor version (4.0.0 version).

Reply:
Timecompression functionality has been also removed? In that case, CNR entry like this could be added:

```
- Remove: time compression support
```